src/payment-service.ts

```diff
@@ -98,7 +102,19 @@ export async function handlePaymentQuerystring(
       'It looks like you are attempting to view a receipt for an unsupported payment.',
     )
   }
-  return await paymentProvider.verifyPaymentTransaction(query)
+
+  const verifiedPaymentTransaction =
+    await paymentProvider.verifyPaymentTransaction(query)
+
+  const schedulingBooking = await getSchedulingBooking()
```

Comment:
hasn't it already been removed by this stage?

Reply:
This is different data.